agents1/OfficialAgent.py

```diff
@@ -813,18 +801,24 @@ def _trustBelief(self, members, trustBeliefs, folder, receivedMessages):
         '''
         # Update the trust value based on for example the received messages
         for message in receivedMessages:
-            # Increase agent trust in a team member that rescued a victim
+            # Increase agent trust in a team member that communicates they will rescue a victim
             if 'Collect' in message:
-                trustBeliefs[self._humanName].competence += 0.10
-                # Restrict the competence belief to a range of -1 to 1
-                trustBeliefs[self._humanName].competence = np.clip(trustBeliefs[self._humanName]['competence'], -1, 1)
+               trustBeliefs[self._humanName].increment_willingness(0.4)
```

Comment:
Also we are still a bit confused about when we should update competence or willingness so this is something we should ask the TAs in the lab, these are just examples to show the trust representation works